homeassistant/components/air_quality/waqi.py

```diff
@@ -27,15 +27,20 @@
 ATTR_HUMIDITY = 'humidity'
 ATTR_PRESSURE = 'pressure'
 
-KEY_TO_ATTR = {
-    'pm25': ATTR_PM_2_5,
-    'pm10': ATTR_PM_10,
-    'h': ATTR_HUMIDITY,
-    'p': ATTR_PRESSURE,
-    't': ATTR_TEMPERATURE,
-    'o3': ATTR_OZONE,
-    'no2': ATTR_NO2,
-    'so2': ATTR_SO2,
+PROP_TO_ATTR = {
+    'air_quality_index': ATTR_AQI,
```

Comment:
Don't duplicate attributes in `device_state_attributes` that are already reported by the base class from the properties.

Reply:
Fixed.